src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.9.0
+12 June, 2023
+
+### Deprecation Notice
+
+- Support for Kubernetes [1.23](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.23.md) has been deprecated and will be removed in the next release.
+- Removed support for dedicated tls secrets for each ingress. 
+
+### Features
```

Comment:
@ifbyol should we mention any changes from lego epic 6 & 7'

Reply:
We should mention that `okteto deploy --remote` is supported now in scenarios with self-signed certificates and private CAs. This is the main change introduced in this chart regrading epic 6.

For epic 7, I wouldn't mention anything as there is an scenario we need to review